services/ui-src/src/components/cards/EntityCard/EntityCardTopSection.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { render, screen } from "@testing-library/react";
+import { EntityCardTopSection } from "./EntityCardTopSection";
+
+jest.mock("utils/state/useStore");
+
+const formattedEntityData = {
+  category: "LTSS-related standard: provider travels to the enrollee",
+  methodFrequency: "Annually",
+  monitoringMethods: [],
+  population: "MLTSS",
+  provider: "Behavioral health",
+  region: "Small counties",
+  standardDescription: "fdsfds",
+  standardType: "Maximum distance to travel",
+};
+
+const entityCardTopSectionComponent = (
+  <EntityCardTopSection
+    entityType="accessMeasures"
+    formattedEntityData={formattedEntityData}
+    printVersion={true}
+  />
+);
+
+describe("Test EntityCardTopSection renders", () => {
+  beforeEach(() => {
+    window.location.pathname === "/mcpar/export";
+  });
+  test("EntityStepCardTopSection renders correctly", () => {
+    render(entityCardTopSectionComponent);
+    expect(screen.getByText("fdsfds")).toBeVisible();
+  });
+
+  test("EntityStepCardTopSection header renders p tag wrapper in report", () => {
```

Comment:
Smol suggestion: maybe this could be renamed `EntityStepCardTopSection header renders p tag wrapper in PDF report` so it's more clear what the test does?